Python/PRP/pyCreatableConvert.cpp

```diff
@@ -842,7 +471,8 @@ PyObject* ICreate(plCreatable* pCre)
         case kSoundMsg: pySoundMsg_FromSoundMsg(plSoundMsg::Convert(pCre));
         default:
             // many messages are not implemented, make sure they are at least a plMessage
-            if (dynamic_cast<plMessage*>(pCre)) return pyMessage_FromMessage(plMessage::Convert(pCre));
+            if (dynamic_cast<plMessage*>(pCre))
+                return pyMessage_FromMessage(plMessage::Convert(pCre));
             return pyCreatable_FromCreatable(pCre);
```

Comment:
Yeah, as I mentioned, there's still a lot more I want to do...  But the branch was sitting around gathering dust for a while, so I thought I'd get what I had out there for now and come back to add more later.